soc/arm/st_stm32/stm32f0/Kconfig.defconfig.stm32f031x6

```diff
@@ -0,0 +1,14 @@
+# ST Microelectronics STM32F031X6 MCU
+
+# Copyright (c) 2021 Sebastian Schwabe <sebastian.schwabe@mailbox.tu-dresden.de>
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_STM32F031X6
+
+config SOC
+	default "stm32f031x6"
+
+config NUM_IRQS
+	default 28
```

Comment:
could you confirm 28 (RM0091 mentioning 32 maskable interrupt channels)

Reply:
I referred to issue #16716 and [this comment](https://github.com/zephyrproject-rtos/zephyr/issues/16716#issuecomment-500679521).  In `modules/hal/stm32/stm32cube/stm32f0xx/soc/stm32f031x6.h` the highest number is 27 so I set 28 for NUM_IRQ. I did not find an exact guide for STM32 chips in the doc so I thought the information in the issue is fine.